src/locales/cy/default.yml

```diff
@@ -97,7 +97,7 @@ validation:
   isUkPostcode: "Rhowch god post yn y DU"
   alphaNumericWithSpecialChars: "Ni ddylai eich {{ label }} gynnwys unrhyw nodau neu symbolau arbennig"
   default: "Mae'n rhaid i chi ateb y cwestiwn hwn"
-  houseNameOrHouseNumber: "Rhowch enw tŷ neu rif tŷ"
+  ukBuildingAddressEmptyValidator: "Rhowch rif y neu enw tŷ neu rhif fflat"
```

Comment:
Hmm not sure this is right `Rhowch rif y neu enw tŷ neu rhif fflat` ins Google translate is `Enter the number or house name or flat number`

Reply:
Changed to `Rhowch rif y neu enw tŷ`